app/controllers/admin_controller.rb

```diff
@@ -1,15 +1,33 @@
 class AdminController < ApplicationController
```

Comment:
Admin is a type of user right? Should we move this code to the user controller and then authenticate that the user is an admin before actually doing these actions in the controller?